server/server/server.js

```diff
@@ -0,0 +1,27 @@
+const prisma = require('./db/db')
+const express = require('express');
+const { PrismaClient } = require('@prisma/client');
+const prisma = new PrismaClient();

```

Comment:
Not needed. You have this same logic extracted in the db/db.js folder so when you want to use it you should import from there